docs/understand/model.md

```diff
@@ -68,13 +66,15 @@ Creating a branch is a **zero-copy operation**; instead of duplicating data, it
 
 ### Tags
 
-Tags are a way to give a meaningful name to a specific commit. 
-Using tags allow users to reference specific releases, experiments or versions by using a human friendly name.
+Tags are a way to give a meaningful name to a specific commit.
+Using tags allow users to reference specific releases, experiments, or versions by using a human friendly name.
```

Comment:
I <3 Oxford commas.

But I still couldn't resist:

![Best Comma, is, Oxford comma](https://github.com/treeverse/lakeFS/assets/7802932/07c9641a-8d63-4d7e-828b-1ce19a60bd43)

https://imgflip.com/i/8bvemm

Reply:
> couldn't resist

Understandable